### PR TITLE
Added support for "language code" (which forces the widget to render in a specific language)

### PR DESCRIPTION
**Pull request by armezit**

### CODE REVIEW

File: Package/Captcha/Captcha.php [hunk 1]
```diff
@@ -255,9 +263,19 @@ public function html()
...
+         */
+        $jsParams = array();
+        if (!is_null($this->languageCode)) {
+            $jsParams['hl'] = $this->languageCode;
+        }
+        $jsParamsQuery = !empty($jsParams) ? '?'.http_build_query($jsParams) : '';
 
         return
-            '<script src="https://www.google.com/recaptcha/api.js" async defer></script>' .
+            '<script src="https://www.google.com/recaptcha/api.js'.$jsParamsQuery.'" async defer></script>' .
```
Comment (AlekseyKorzun) [inline Package/Captcha/Captcha.php]: Please wrap the $jsParamsQuery in spaces per PSR.

File: Package/Captcha/Captcha.php [hunk 2]
```diff
@@ -410,6 +428,28 @@ public function setSize($size)
...
+    public function setLanguageCode($languageCode)
+    {
+        $this->languageCode = $languageCode;
+        return $this;
+    }
+    
+    /**
+     * Get widget language code
+     * @return string
+     * @see https://developers.google.com/recaptcha/docs/language
```
Comment (AlekseyKorzun) [inline Package/Captcha/Captcha.php]: Please move @see above @return for consistency sake. There should be a space between description and phpDoc blocks. Appreciate it.

File: Package/Captcha/Captcha.php [hunk 3]
```diff
@@ -106,6 +106,14 @@ class Captcha
...
      */
     protected $size = 'normal';
+    
+    /**
+     * Forces the widget to render in a specific language. Auto-detects the user's language if unspecified.
+     * 
+     * @var string
+     * @see https://developers.google.com/recaptcha/docs/language
+     */
+    protected $languageCode = null;
```
Comment (AlekseyKorzun) [inline Package/Captcha/Captcha.php]: Please move @see above @var if possible. You don't need to set = null; can get rid of that.

File: Package/Captcha/Captcha.php [hunk 4]
```diff
@@ -255,9 +263,19 @@ public function html()
...
+        
+        /**
+         * JavaScript resource (api.js) parameters
+         * @see https://developers.google.com/recaptcha/docs/display#js_param
+         */
+        $jsParams = array();
+        if (!is_null($this->languageCode)) {
+            $jsParams['hl'] = $this->languageCode;
+        }
+        $jsParamsQuery = !empty($jsParams) ? '?'.http_build_query($jsParams) : '';
```
Comment (AlekseyKorzun) [inline Package/Captcha/Captcha.php]: Add (string) prior to !empty.

**Comment by AlekseyKorzun** [comment]
Hey @armezit sorry for the delay.

I appreciate your contribution! Please look at my comments and let me know what you think. 

Cheers!

File: Package/Captcha/Captcha.php (hunk 4 above)
Comment (armezit) [inline Package/Captcha/Captcha.php]: I think cast to string is not required, as it always returns string: either if $jsParams is empty or not.

File: Package/Captcha/Captcha.php (hunk 1 above)
Comment (armezit) [inline Package/Captcha/Captcha.php]: Do you mean something like this?
```php
return
            '<script src="https://www.google.com/recaptcha/api.js '.$jsParamsQuery.' " async defer></script>' .
```
I'm not very familiar with PSR standards :)

File: Package/Captcha/Captcha.php (hunk 2 above)
Comment (armezit) [inline Package/Captcha/Captcha.php]: @AlekseyKorzun Done. But some other methods (like: setTheme(), setSize(), setType()) have the @see above @return. Actually I wrote phpDoc of the setLanguageCode() according to them.

File: Package/Captcha/Captcha.php (hunk 1 above)
Comment (AlekseyKorzun) [inline Package/Captcha/Captcha.php]:
```php
return
            '<script src="https://www.google.com/recaptcha/api.js ' . $jsParamsQuery . ' " async defer></script>' .
```

File: Package/Captcha/Captcha.php (hunk 4 above)
Comment (AlekseyKorzun) [inline Package/Captcha/Captcha.php]: @armezit The http_build_query could return something else in future / previous PHP versions this is simply a safe guard for possible revisions to PHP core. Also '?'.http should be '?' . http_

Thank you!

File: Package/Captcha/Captcha.php (hunk 2 above)
Comment (AlekseyKorzun) [inline Package/Captcha/Captcha.php]: @armezit  Sorry if that's the case, it's my fault.  Will correct to make sure they are in sync!

**Comment by armezit** [comment]
@AlekseyKorzun I fixed the issues

**Comment by AlekseyKorzun** [comment]
@armezit :+1:  

Thanks brother!

**Pull request closed without merging**